packages/mds-types/index.ts

```diff
@@ -1,426 +1,634 @@
-/*
```

Comment:
What's going on here? The whole file obviously didn't change. Is this some CR/LF because someone edited this on a Windows machine that didn't respect editorconfig, etc.?

Reply:
That is precisely what happened, we have a fix in a PR targeting this branch that we will merge in ASAP